powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2024 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.app.powerauthfido2demo;
```

Comment:
The package name seems quite long, can we either simplify it or split into two subfolders?

Reply:
set to `com.wultra.app.powerauth.fido2.demo`